Flipper/Packages/UI/Sources/Device/UpdateCard/Components/UpdateWhatsNewButton.swift

```diff
@@ -63,7 +58,6 @@ struct UpdateWhatsNewButton: View {
 }
 
 struct WhatsNewScreen: View {
```

Comment:
Whoud be better to have separate file for the screen and `fullScreenCover` within `DeviceUpdateCard` just after `.fullScreenCover(isPresented: $updateModel.showUpdate)`